app/webpack/packs/application.js

```diff
@@ -10,11 +10,12 @@ import TurbolinksAdapter from 'vue-turbolinks';
 import AuthApp from '../components/authentication/app.vue';
 
 Rails.start();
-Vue.use(TurbolinksAdapter);
 Turbolinks.start();
 
+Vue.use(TurbolinksAdapter);
+
 document.addEventListener('turbolinks:load', () => {
-  const element = document.getElementById('application');
+  const element = document.getElementById('sign_in');
```

Comment:
So are we going to have different Vue app instances for different features? In other projects, I have just gone with one global project called `application` that only loads components if the page has it defined.

Reply:
Yeah, I know what you mean - I usually do the same. In this particular case I expect (currently) we'll only have a handful of Vue components, and only active on certain pages, so it seemed more relevant to only bootstrap in those cases.